membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -353,7 +353,7 @@ class AccountController(commonActions: CommonActions, override val controllerCom
       email = accountSummary.billToContact.email,
       paymentDetails = upToDatePaymentDetails,
       stripePublicKey = stripeService.publicKey,
-      accountHasMissedRecentPayments = accountHasMissedPayments(subscription.accountId, freeOrPaidSub.isRight, accountSummary.invoices, accountSummary.payments),
+      accountHasMissedRecentPayments = if (freeOrPaidSub.isRight) accountHasMissedPayments(subscription.accountId, accountSummary.invoices, accountSummary.payments) else false,
```

Comment:
maybe you could write this as :
```scala
      accountHasMissedRecentPayments = freeOrPaidSub.isRight && accountHasMissedPayments(subscription.accountId, accountSummary.invoices, accountSummary.payments) 

```

Reply:
Yes that's much nicer, thanks for the suggestion